src/olympia/activity/models.py

```diff
@@ -577,13 +570,18 @@ def to_string(self, type_=None):
                 ):
                     validation = 'ignored'
 
-                file_ = self.f(
+                file_ = format_html(
                     '<a href="{0}">{1}</a> (validation {2})',
-                    arg.get_absolute_url(),
+                    get_absolute_url(arg),
                     arg.pretty_filename,
                     validation,
                 )
                 arguments.remove(arg)
+            if isinstance(arg, UserProfile) and not user:
```

Comment:
`{user}` in format strings used to be the user that the `ActivityLog` was attached to. In other words, the user that did the activity. But in several cases the format string was incorrect, we expected it to be the target of the activity, which is not the same thing. For example, for `ADMIN_USER_EDITED`, `{user}` was the admin, and the user being edited was actually `{0}`.

Since this was super confusing and not aligned with the way the rest of the arguments are handled, instead I made `{user}` be treated like a regular argument, introduced `{user_responsible}` which is the user that did the activity, and fixed all the format strings.